lms/djangoapps/teams/static/teams/js/views/teams_tab.js

```diff
@@ -380,24 +380,18 @@
                 createTeamsListView: function(options) {
                     var topic = options.topic,
                         collection = options.collection,
-                        myTopicTeamsCollection = new TeamCollection(
-                            this.context.userInfo.teams,
-                            {
-                                teamEvents: this.teamEvents,
-                                course_id: this.context.courseID,
-                                username: this.context.userInfo.username,
-                                topic_id: topic.get('id'),
-                                perPage: 5,
-                                parse: true,
-                                url: this.context.myTeamsUrl
+                        topicTeams = _.filter(
+                            this.context.userInfo.teams.results,
+                            function(team) {
```

Comment:
I'm not positive but I think you actually have to stick with what you have originally. `In teams/views.py::TeamsDashboardView.get` when we're creating userInfo, we paginate with `MyTeamsPagination`, which has a pagination size of 5, which I think means that if a user is on more than 5 teams, this list won't include some of their teams